pkg/monitors/statuscake/statuscake-monitor.go

```diff
@@ -199,9 +208,15 @@ func buildUpsertForm(m models.Monitor, cgroup string) url.Values {
 	if providerConfig != nil && providerConfig.Confirmation > 0 {
 		f.Add("confirmation", strconv.Itoa(providerConfig.Confirmation))
 	}
-	if providerConfig != nil {
+	if providerConfig != nil && len(providerConfig.FindString) > 0 {
 		f.Add("find_string", providerConfig.FindString)
 	}
+	if providerConfig != nil && len(providerConfig.RawPostData) > 0 {
+		f.Add("post_raw", providerConfig.RawPostData)
+	}
```

Comment:
Should this be here?
I see this change is also a part of your #609 PR

Reply:
You are correct! This is an artifact of splitting my commits into different PRs. I have fixed it now :)